charts/ceph-csi-cephfs/templates/provisioner-deployment.yaml

```diff
@@ -55,6 +55,7 @@ spec:
 {{- if .Values.topology.enabled }}
             - "--feature-gates=Topology=true"
 {{- end }}
+            - "--default-fstype=ext4"
```

Comment:
provide an option as in https://github.com/ceph/ceph-csi/pull/1771/commits/47faf88e51bdd74089b93640afe35c6bde62481d. I will drop that commit from #1771

Reply:
That's fine I can clsoe this PR